docs/guides/aerogear-android/push.asciidoc

```diff
@@ -89,100 +116,46 @@ You can also add a default message handler to the receiver in AndroidManifest.xm
 
 ----
 
-==== Registering with GCM and Unified Push Server
+==== Handling messages
 
-This code example demonstrates how to create a link:/docs/specs/aerogear-android/org/jboss/aerogear/android/unifiedpush/PushConfig.html[PushConfig], provide values required to register the application with GCM and Unified Push, and perform the registration.
+Currently, push messages are received by an instance of link:/docs/specs/aerogear-android-push/org/jboss/aerogear/android/unifiedpush/gcm/AeroGearGCMMessageReceiver.html[AeroGearGCMMessageReceiver]. They are processed the messages and passed to link:/docs/specs/aerogear-android-push/org/jboss/aerogear/android/unifiedpush/RegistrarManager.html[RegistrarManager] via notifyHandlers method.
```

Comment:
"The messages are processed and passed to RegistrarManager via the notifyHandlers method" sounds better